client-app/router/routes/account.ts

```diff
@@ -119,8 +119,7 @@ export const accountRoutes: RouteRecordRaw[] = [
     name: "Notifications",
     component: Notifications,
     beforeEnter(_to, _from, next) {
-      const { isActive } = usePushMessages();
-      if (themeContext.value?.settings?.push_messages_enabled && isActive) {
+      if (isPushMessagesActive.value) {
         next();
       } else {
         next({ name: "Dashboard" });
```

Comment:
left this return to Dashboard page, since there are 2 similar redirects in that file, so to keep consistency didn't touch that.